corehq/apps/hqpillow_retry/static/hqpillow_retry/js/pillow_errors.js

```diff
@@ -0,0 +1,8 @@
+hqDefine("hqpillow_retry/js/pillow_errors", function() {
+    $(function() {
+        $(document).on('click', '#check_all', function() {
+            var oTable = $('#report_table_' + hqImport('hqwebapp/js/initial_page_data').get('slug')).dataTable();
```

Comment:
this might be easier to read with slug assigned to a var outside of the concatenation